crates/net/network/src/transactions/validation.rs

```diff
@@ -0,0 +1,555 @@
+//! Validation of [`NewPooledTransactionHashes68`] entries. These are of the form
+//! `(ty, hash, size)`. Validation and filtering of entries is network dependent. [``]
+
+use core::fmt;
+use std::{collections::HashMap, mem};
+
+use derive_more::{Deref, DerefMut, Display};
+use itertools::izip;
+use reth_eth_wire::{
+    NewPooledTransactionHashes66, NewPooledTransactionHashes68, ValidAnnouncementData,
+};
+use reth_primitives::{Signature, TxHash, TxType};
+use tracing::{debug, trace};
+
+/// The size of a decoded signature in bytes.
+pub const SIGNATURE_DECODED_SIZE_BYTES: usize = mem::size_of::<Signature>();
+
+/// Max encoded length of transaction types pre-blob transaction type.
+pub const MAX_ENCODED_PRE_BLOB_TX_LENGTH: usize = usize::pow(2, 17);
```

Comment:
this is per blob yes, we can also use the constants from primitives
```suggestion
pub const MAX_ENCODED_PER_BLOB_TX_LENGTH: usize = usize::pow(2, 17);
```

Reply:
I was going for pre-blob, since if the max encoded length of the blob tx changes then that constant is highly misleading. will call it `max_encoded_tx_length_anscestors_of_blob_tx_type`